tap_salesforce/salesforce/__init__.py

```diff
@@ -199,6 +206,7 @@ def field_to_property_schema(field, mdata): # pylint:disable=too-many-branches
 
     return property_schema, mdata
 
+#pylint: disable=too-many-positional-arguments
```

Comment:
I was skimming this PR and thought it was weird to disable a linter for the whole class. So I just wanted to note that this class level disable is to fix things for just two methods.

Here's a `pylint` run after deleting this line:
```
$ pylint tap_salesforce -d 'broad-except,broad-exception-caught,broad-exception-raised,chained-comparison,empty-docstring,fixme,invalid-name,line-too-long,missing-class-docstring,missing-function-docstring,missing-module-docstring,no-else-raise,no-else-return,too-few-public-methods,too-many-arguments,too-many-branches,too-many-lines,too-many-locals,ungrouped-imports,wrong-spelling-in-comment,wrong-spelling-in-docstring,consider-using-f-string,logging-fstring-interpolation,wrong-import-order,stop-iteration-return,logging-format-interpolation'
************* Module tap_salesforce.salesforce
tap_salesforce/salesforce/__init__.py:211:4: R0917: Too many positional arguments (12/5) (too-many-positional-arguments)
tap_salesforce/salesforce/__init__.py:292:4: R0917: Too many positional arguments (7/5) (too-many-positional-arguments)

-------------------------------------------------------------------
Your code has been rated at 9.98/10 (previous run: 10.00/10, -0.02)
```

And here's the changes to get this back to a 10/10 rating
```diff
diff --git a/tap_salesforce/salesforce/__init__.py b/tap_salesforce/salesforce/__init__.py
index f144b53..2a334cc 100644
--- a/tap_salesforce/salesforce/__init__.py
+++ b/tap_salesforce/salesforce/__init__.py
@@ -206,9 +206,8 @@ def field_to_property_schema(field, mdata): # pylint:disable=too-many-branches

     return property_schema, mdata

-#pylint: disable=too-many-positional-arguments
 class Salesforce():
-    # pylint: disable=too-many-instance-attributes,too-many-arguments
+    # pylint: disable=too-many-instance-attributes,too-many-arguments,too-many-positional-arguments
     def __init__(self,
                  refresh_token=None,
                  token=None,
@@ -284,7 +283,7 @@ class Salesforce():
                                                                        self.quota_percent_per_run)
             raise TapSalesforceQuotaExceededException(partial_message)

-    # pylint: disable=too-many-arguments
+    # pylint: disable=too-many-arguments,too-many-positional-arguments
     @backoff.on_exception(backoff.expo,
                           (requests.exceptions.ConnectionError, requests.exceptions.Timeout),
                           max_tries=10,
```

Maybe we should open a separate issue to clean up some of these inline disables. For example, `too-many-arguments` appears in `$PYLINT_DISABLE_LIST` and doesn't need to be inlined anymore

Reply:
Moved these class level inline pylint disables to function level. Created separate ticket to clean these inline disables - [TDL-26624](https://jira.talendforge.org/browse/TDL-26624)